cg/apps/demultiplex/sample_sheet/sample_sheet_creator.py

```diff
@@ -221,35 +176,9 @@ def __init__(
         if flow_cell.bcl_converter == BclConverter.BCL2FASTQ:
             raise SampleSheetError(f"Can't use {BclConverter.BCL2FASTQ} with sample sheet v2")
 
-    def update_barcode_mismatch_values_for_samples(
-        self, samples: list[FlowCellSampleBCLConvert]
-    ) -> None:
-        """Update barcode mismatch values for both indexes of given samples."""
-        for sample in samples:
-            update_barcode_mismatch_values_for_sample(
-                sample_to_update=sample, samples_to_compare_to=samples
-            )
-
-    def add_override_cycles_to_samples(self) -> None:
-        """Add override cycles attribute to samples."""
-        read1_cycles: str = f"Y{self.run_parameters.get_read_1_cycles()};"
-        read2_cycles: str = f"Y{self.run_parameters.get_read_2_cycles()}"
-        length_index1: int = self.run_parameters.get_index_1_cycles()
-        length_index2: int = self.run_parameters.get_index_2_cycles()
-        for sample in self.lims_samples:
-            index1_cycles: str = f"I{length_index1};"
-            index2_cycles: str = f"I{length_index2};"
-            sample_index1_len: int = len(get_index_pair(sample)[0])
-            sample_index2_len: int = len(get_index_pair(sample)[1])
-            if sample_index1_len < length_index1:
-                index1_cycles = f"I{sample_index1_len}N{length_index1 - sample_index1_len};"
-            if sample_index2_len < length_index2:
-                index2_cycles = (
-                    f"N{length_index2-sample_index2_len}I{sample_index2_len};"
-                    if self.index_settings.are_i5_override_cycles_reverse_complemented
-                    else f"I{sample_index2_len}N{length_index2 - sample_index2_len};"
-                )
-            sample.override_cycles = read1_cycles + index1_cycles + index2_cycles + read2_cycles
+    def remove_unwanted_samples(self) -> None:
+        """Filter out samples with single indexes."""
+        LOG.info("Removing of single index samples is not required for V2 sample sheet")
```

Comment:
As it is defined as an abstract method in the parent class it has to exist in the child classes. This will be removed when the Bcl2Fast class is deprecated

Reply:
👍